server/server.go

```diff
@@ -510,7 +505,10 @@ func New(config *Config) (*Server, error) {
 		return nil, err
 	}
 
-	containerServer := libkpod.New(r, store, imageService, config.SignaturePolicyPath)
+	containerServer, err := libkpod.New(libkpod.DefaultConfig())
+	if err != nil {
```

Comment:
The DefaultConfig() will provide some of those, others can be generated from the information within it